RetroBar/Themes/Windows 95-98.xaml

```diff
@@ -20,5 +20,11 @@
 
     <ToolTip x:Key="StartButtonTip" Content="{DynamicResource start_button_tip_98}" />
 
+    <Style TargetType="ToggleButton"
+           x:Key="StartButton"
+           BasedOn="{StaticResource StartButton}">
+        <Setter Property="FocusVisualStyle" Value="{StaticResource ClassicFocusVisual}" />
```

Comment:
Nice, since only 9x uses this, and others should not have the focus style, I think setting this to `{x:Null}` in System.xaml, Watercolor.xaml, and Windows XP Blue.xaml would be good.

Reply:
I will change it, on the other systems this is found if the "Underline keyboard shortcuts and access keys" option is turned on, by default it is turned off, so I guess there shouldn't be a problem, this should also fix #133.